ros/src/tl_detector/tl_detector.py

```diff
@@ -41,21 +47,30 @@ def __init__(self):
         self.upcoming_red_light_pub = rospy.Publisher('/traffic_waypoint', Int32, queue_size=1)
 
         self.bridge = CvBridge()
-        self.light_classifier = TLClassifier()
```

Comment:
please revert.

Reply:
Shall be reverted after the TrafficLight classifier is implemented.

Reply:
Reverted.